lib/ZwaveClient.js

```diff
@@ -107,8 +107,26 @@ function ZwaveClient (config, socket) {
 
   config.networkKey = config.networkKey || process.env.NETWORK_KEY
 
-  this.nodes = []
+  this.nodes = new Map()
   this.storeNodes = jsonStore.get(store.nodes)
+
+  // convert store nodes from array to object
+  if (Array.isArray(this.storeNodes)) {
+    const storeNodes = {}
+
+    for (const n of this.storeNodes) {
+      if (n) {
+        this.storeNodes[n.id] = n
+      }
+    }
+
+    this.storeNodes = storeNodes
+
+    jsonStore.put(store.nodes, storeNodes).catch(err => {
+      logger.error('Error while updating store nodes', err)
+    })
+  }
```

Comment:
I don't really see the point of this.

Reply:
It would work in both ways, was just for consistency

Reply:
Nope I really need this